src/config.rs

```diff
@@ -432,6 +437,7 @@ pub enum GooseDefault {
     WebSocketHost,
     /// An optional default for port WebSocket Controller listens on.
     WebSocketPort,
+    AcceptInvalidCerts,
```

Comment:
Please add a comment, similar to the above:
` An optional default for not validating https certificates.`

Reply:
Thanks!